docs/getting-started/quickstart.md

```diff
@@ -7,7 +7,9 @@ import TabItem from '@theme/TabItem';
 
 # DBOS Quickstart
 
-This guide shows you how to deploy your first app to DBOS Cloud in less than 5 minutes. After that, it shows you how to develop DBOS apps locally.
+You start developing a DBOS app by creating a folder for it on your computer. From this folder, you can run the app locally and deploy to DBOS Cloud in minutes.
```

Comment:
We need to iterate a lot on this opening wording, I don't think it's great now. Here's my suggestion, which still isn't amazing:

> Let's build our first DBOS app!  If you build an app with DBOS, you can run it locally connected to your local Postgres database, or deploy it to the cloud connected to a cloud Postgres database. In this guide, we'll show you how to do both.

Reply:
I know what you mean. I put like 2 hours into that sentence already. And no I don't like yours as much yet :) I think this beginning should be simple....

I'll think about it some more.

Reply:
Should be all set now